src/index.html

```diff
@@ -11,7 +11,147 @@
+          type="text"
+          name="name"
+          autocomplete="off"
+          maxlength="10" minlength="3">
+        </label>
+
+        <label class="form__field">
+          How old are You?:
+          <input
+          type="number"
+          max="100" min="1"
+          value="12"
+          name="age">
+        </label>
+
+        <label class="form__field">
+          Full date of birth:
+          <input
+          type="date"
+          name="dateofbirth">
+        </label>
+
+        <label class="form__field">
+          I accept the term of agreement:
+          <input
+          type="checkbox"
+          name="agreement">
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+
+        <label class="form__field">
+          E-mail:
+          <input
+          type="email"
+          name="email"
+          placeholder="email@example.com">
+        </label>
+
+        <label class="form__field">
+          Password:
+          <input
+          type="password"
+          name="password">
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+
+        <label class="form__field">
+          Do you love cats?
+          <label>
+            <input
+            type="radio"
+            name="love"
+            value="yes">
+            Yes
+          </label>
+
+          <label>
+            <input type="radio"
+            name="love"
+            value="no">
+            No
+          </label>
+        </label>
+
+        <label class="form__field">
+          What is your favorite color?
+          <input type="color"
+          name="color">
+        </label>
+
+        <label class="form__field">
+          What time do you go to bed?
+          <input
+          type="time"
+          name="bed time">
+        </label>
+
+        <label class="form__field">
+          What are your favorite brand of cars?
+          <select name="cars" required multiple>
+            <option value="car">BMW</option>
+            <option value="car">Audi</option>
+            <option value="car">Lada</option>
+          </select>
+        </label>
+
+        <label class="form__field">
+          How do you rate our work?
+          <input
+          type="range"
+          min="1" max="10"
+          name="range"
+          step="1" >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Additional info:</legend>
+
+        <label class="form__field">
+          Comments:
+          <textarea name="comments"></textarea>
+        </label>
+
+        <label class="form__field">
+          Would you recommend us?
+          <select name="recommend">
+            <option value="yes">yes</option>
+            <option value="no">no</option>
+          </select>
+        </label>
+      </fieldset>
+
+      <label class="form__field">
+        <input type="submit">
```

Comment:
It should be the button.
```suggestion
        <button type="submit">Submit</button>
```
Without including inside the tag `label`